content/why/_index.md

```diff
@@ -96,10 +96,85 @@ There should be a self reinforcing energy to all of this: we want tools that wor
 Strategy
 --------
 
+### familiar tools
+
+Specifically, the primary target and scope can be summarized as “must support linux-like systems” and “must pass state primarily by familiar ‘file’-like abstractions”.
+(In other words -- “it should taste like containers” -- this is a proven path to successful adoption that lets others use what they know and pivot towards our tooling incrementally.  Incremental is good.)
+This scope may extend to more systems in the future, but will start here.
+
+### anything not everything
+
+The Timeless Stack tools need to support *anything* -- anything you want to put in them, with no special consideration required -- rather than supporting "everything", but needing plugins to do it.  This stands in contrast to most other build tools.
+
 ### breaking down the problem
 
+There are five major sets of problems that need to be addressed:
+
+1. Framing and hermetic execution and fileset-slinging core
+2. Coordination systems for connecting graphs of executions and outputs -- both for authoring graphs of intent, and then precipitating graphs of actual outcome (which can be repeated).
```

Comment:
Would calling this "pipelines" be useful here? I feel it's a good word that people should know in the context of existing CI systems already in use.